app/renderer/js/utils/custom-config-util.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+const JsonDB = require('node-json-db');
+const Logger = require('./logger-util');
+
+const logger = new Logger({
+	file: 'custom-config-util.log',
+	timestamp: true
+});
+
+let instance = null;
+let dialog = null;
+let app = null;
+
+/* To make the util runnable in both main and renderer process */
+if (process.type === 'renderer') {
+	const remote = require('electron').remote;
+	dialog = remote.dialog;
+	app = remote.app;
+} else {
+	const electron = require('electron');
+	dialog = electron.dialog;
+	app = electron.app;
+}
+
+class CustomConfigUtil {
+	constructor() {
+		if (instance) {
+			return instance;
+		} else {
+			instance = this;
+		}
+
+		this.reloadDB();
+		return instance;
+	}
+
+	getConfigItem(key, defaultValue = null) {
+		try {
+			this.db.reload();
+		} catch (err) {
+			logger.error('Error while reloading custom_settings.json: ');
+			logger.error(err);
+		}
+		const value = this.db.getData('/')[key];
+		if (value === undefined) {
+			return defaultValue;
+		} else {
+			return value;
+		}
+	}
+	// This function returns whether a key exists in the configuration file (custom_settings.json)
+	isConfigItemExists(key) {
+		try {
+			this.db.reload();
+		} catch (err) {
+			logger.error('Error while reloading custom_settings.json: ');
+			logger.error(err);
+		}
+		const value = this.db.getData('/')[key];
+		return (value !== undefined);
+	}
+
+	removeConfigItem(key) {
```

Comment:
@kanishk98 I don't think we will be able to use this function since we won't have rights to remove/delete the data from that particular directory. cc, @priyank-p.

Reply:
Okay, makes sense. I'll remove this method.

Reply:
Also, I don't think we will need the `node-json-db` here. We just need to attempt to read the json which you can do with with reading the file with `fs` and `JSON.parse`.